docs/UserGuide.md

```diff
@@ -9,21 +9,21 @@
 
 0. Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
    > Having any Java 8 version is not enough. <br>
-   This app will not work with earlier versions of Java 8.
+   This application will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds. 
+1. Download the latest `taskell.jar` from here.
+2. Copy the file to the folder you want to use as the home folder for your task manager.
+3. Double-click the file to start the application. The GUI should appear in a few seconds. 
    > <img src="images/Ui.png" width="600">
 
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
+    > ![gui listToday](gif/listToday.gif)<br>
+   e.g. typing **`list today`** and pressing <kbd>Enter</kbd> will list the tasks of today.
 5. Some example commands you can try:
    * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
+   * **`add`** `add buy MA1101R textbook today` : adds a task called �buy MA1101R textbook� to be done by today
```

Comment:
That is not 2 add... 1 is saying what command and the other is an example of how to use the command,